resources/css/index.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Yeseva One";
+  src: url(../fonts/YesevaOne.otf);
+}
+
+@font-face {
+  font-family: "Lato";
+  src: url(../fonts/Lato.ttf);
+}
+body, html {
+  margin: 0;
+  padding: 0;
+}
+.wrapper {
+  margin: 0 auto;
+  width: 1600px;
+}
+
+.container h1 {
+  font-family: "Yeseva One", sans-serif;
```

Comment:
Sort properties alphabetically

Reply:
`p` - also tag, call it may be `.header`, as context goes

Reply:
fixed